src/java.base/share/classes/sun/util/resources/LocaleData.java

```diff
@@ -246,15 +245,15 @@ protected String toOtherBundleName(String baseName, String bundleName, Locale lo
     private static class LocaleDataStrategy implements Bundles.Strategy {
         private static final LocaleDataStrategy INSTANCE = new LocaleDataStrategy();
         // TODO: avoid hard-coded Locales
-        private static Set<Locale> JAVA_BASE_LOCALES
+        private final static Set<Locale> JAVA_BASE_LOCALES
```

Comment:
Canonical modifier order is `static final`

Reply:
It turns out that my IDE was configured NOT to highlight missorted modifiers. Once I reconfigured it, I saw these cases and some other in that same file. I'll fix them all if that's okay.

My recollection is that there have been campaigns on using "the blessed modifiers order". It might be that since the last such crusade the modifiers have gone out of hand, and we might need to re-bless them :-)

Reply:
Fixed in 2b7e0c6.